spec/abstractops.html

```diff
@@ -950,15 +957,12 @@ <h1>ISO 8601 grammar</h1>
           TimeZoneIANANameComponent `/` TimeZoneIANANameTail
 
       TimeZoneIANAName :
-          TimeZoneIANANameTail
-
-      TimeZoneBracketedName :
-          TimeZoneIANAName
-          `Etc/GMT` ASCIISign Hour
+          `Etc/GMT` ASCIISign UnpaddedHour
           TimeZoneUTCOffsetName
+          TimeZoneIANANameTail
```

Comment:
While this grammar is changing, we should resolve its ambiguity as well by prohibiting TimeZoneIANANameTail from matching an Etc/GMT… zone (it might not matter in practice, but allowing such ambiguity just seems like bad form).
```suggestion
          TimeZoneUTCOffsetName
          `Etc/GMT` ASCIISign UnpaddedHour
          TimeZoneIANANameTail but not `Etc/GMT` ASCIISign UnpaddedHour
```